rotkehlchen/tests/unit/decoders/test_makerdao.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import deserialize_evm_tx_hash
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0x648aA14e4424e0825A5cE739C8C68610e143FB79']])  # noqa: E501
+def test_makerdao_simple_transaction(
+        database,
+        ethereum_manager,
+        function_scope_messages_aggregator,
+):
+    """Data taken from
+    https://etherscan.io/tx/0x95de47059bcc084ebb8bdd60f48fbcf05619c2af84bf612fdc27a6bbf9b5097e
+    """
+    tx_hash = deserialize_evm_tx_hash('0x95de47059bcc084ebb8bdd60f48fbcf05619c2af84bf612fdc27a6bbf9b5097e')  # noqa: E501
+    # We don't need any events here, we just check that no errors occure during decoding
```

Comment:
```suggestion
    # We don't need any events here, we just check that no errors occur during decoding
```

That's lazy though @nebolax.

You should check that the events are correct. not just check that there is no errors.